### DISCUSSION TITLE
Disentangling failed: 'Incomplete/Complicated graph: please check around EDGE_13240883!'

### DISCUSSION THREAD

**Discussion by curtainDian** [Kinggerm/GetOrganelle#172] [Q&A]
First of all，I want to say getorganelle is an excellent software! It is almost the best software of assembling chloroplast I've ever seen.

I tried default settings and then adjusted parameters for many times ( such as lower word size, denser and higher kmer gradient and more rounds). However, I still got the incomplete chloroplast. It is the following command I tried :
get_organelle_from_reads.py -1 /ifs1/Work/wusl/Rosa_chloroplast/00.database/yc-24_R1.fq.gz -2 /ifs1/Work/wusl/Rosa_chloroplast/00.database/yc-24_R2.fq.gz -t 4 -o yc-24-6 -s /ifs1/Work/wusl/Rosa_chloroplast/03.GetOrganelle_assemble/analysis/yc-11/0.0.1/SeedDatabase/Rosa_OB_chloroplast.fasta -F embplant_pt -R 20 -k 21,39,45,65,85,105,127 -w 75

and the get_org_log.txt are as follows.

Thanks!

[get_org.log.txt](https://github.com/Kinggerm/GetOrganelle/files/8704858/get_org.log.txt)

**Comment by Kinggerm** [comment]
This is a typical problem for a common dataset and not an issue. So it was moved to Discussion.

Have you tried increasing the #reads using `--max-reads` and/or `--reduce-reads-for-coverage`?

**Comment by curtainDian** [comment]
Thanks！ I did not use the --max-reads or --reduce-reads-for-coverage before . And when I used one of parameters above, error appeared: "get_organelle_from_reads.py: error: argument --reduce-reads-for-coverage: expected one argument". Could you please tell me how to solve it ? I used one of the examples from given : ###  Embryophyta plant plastome, 2*(1G raw data, 150 bp) reads
get_organelle_from_reads.py -1 sample_1.fq -2 sample_2.fq -s cp_seed.fasta -o plastome_output  -R 15 -k 21,45,65,85,105 -F embplant_pt and added --reduce-reads-for-coverage to commands but it still did not work .

**Reply by Kinggerm** [reply]
You should definitely try `--reduce-reads-for-coverage` and/or `--max-reads`, which is one of the top three options stated in the [FAQ](https://github.com/Kinggerm/GetOrganelle/wiki/FAQ#what-should-i-do-with-incomplete-resultbroken-assembly-graph)

Here the `expected one argument` error means that `--reduce-reads-for-coverage` takes a parameter. Also, see the [usage](https://github.com/Kinggerm/GetOrganelle/wiki/Usage). You need to assign a number for `--reduce-reads-for-coverage`. While the default is `500`, you are recommended to specify a larger number, such as `--reduce-reads-for-coverage 1000`.

**Reply by liuqing0518** [reply]
Hello, I encountered the same problem as you did. I wonder if your issue has been resolved?
